### ISSUE TITLE
Storage adapter config

### ISSUE THREAD

**Issue by pgporada** [caddyserver/caddy#2767]
```
# cat /etc/caddy/Caddyfile.v2 
{
    storage file_system {
        root /etc/ssl/caddy
    }
}
```

```
# caddy adapt-config  --input /etc/caddy/Caddyfile.v2 --adapter caddyfile --pretty
{
	"storage": {
		"root": "/etc/ssl/caddy"
	},
```

**Comment by mholt** [comment]
Oh, thanks for filing an issue about this Phil! :smile: Fixed in 735d6ce405c353b5a9f65ead769d5e7319fe71ce.

**Comment by pgporada** [comment]
Confirmed fixed!
```
$ caddy adapt-config  --input /etc/caddy/Caddyfile.v2 --adapter caddyfile --pretty
{
	"storage": {
		"module": "file_system",
		"root": "/etc/ssl/caddy"
	},
```
